dnsapi/dns_regru.sh

```diff
@@ -92,9 +92,10 @@ _get_root() {
   domains_list=$(echo "${response}" | grep dname | sed -r "s/.*dname=\"([^\"]+)\".*/\\1/g")
 
   for ITEM in ${domains_list}; do
+    IDN_ITEM="$(echo "${ITEM}" | idn)"
```

Comment:
don't use `idn`,  please use `_idn()` function instead.

Reply:
Hello! Cant use this now, can't check dns record, because it is valid, dns verification expires 27 december) I Check it later.

Reply:
> 

How to use `_idn()`? 
```suggestion
    IDN_ITEM="$(echo "${ITEM}" | idn)"
```

Reply:
Hello, I did it!

`    IDN_ITEM="$(_idn "${ITEM}")"
`